setup.py

```diff
@@ -1,12 +1,31 @@
 from setuptools import find_namespace_packages, setup
 
-__version__ = "0.1.0"
+__version__ = "0.1.1"
 
 setup(
     name="liger_kernel",
     version=__version__,
+    description="Efficient Triton kernels for LLM Training",
+    long_description=open("README.md").read(),
+    long_description_content_type="text/markdown",
+    license="BSD-2-Clause",
+    url="https://github.com/linkedin/Liger-Kernel",
     package_dir={"": "src"},
     packages=find_namespace_packages(where="src"),
+    classifiers=[
+        'Development Status :: 4 - Beta',
+        'Intended Audience :: Developers',
```

Comment:
where do we get these?

Reply:
From here: https://pypi.org/classifiers/